packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -0,0 +1,69 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+@layer components {
+  .vtmn-text-input,
+  .vtmn-text-input::before,
+  .vtmn-text-input::after {
+    box-sizing: border-box;
+  }
+
+  .vtmn-text-input {
+    box-shadow: inset 0 0 0 1px var(--vtmn-color_grey-dark-3);
+
+    @apply vtmn-block
+      vtmn-font-normal
+      vtmn-text-base
+      vtmn-text-black
+      vtmn-p-3
+      vtmn-mt-2
+      vtmn-mb-1
+      vtmn-leading-none
+      vtmn-bg-white
+      vtmn-rounded;
+  }
+
+  .vtmn-text-input:not([disabled]):hover {
+    box-shadow: inset 0 0 0 1px var(--vtmn-color_brand-digital);
+  }
+
+  .vtmn-text-input:not([disabled]):focus,
+  .vtmn-text-input:not([disabled]):active {
+    outline: none;
```

Comment:
Removing outline is not a good accessibility practice... 

It should be kept and designed properly by designers. IMHO this element is quite tricky to tweak. I think designers will have to adapt to the difficulty of implementation

Reply:
But we have our proper outline.Like we did with the button.
How can you style outline with border radius?
I only find the trick of box-shadow inset...

Reply:
Maybe we can report it to UX Team ? 
I am agree with @thollander, HTML / CSS Standards > ALL ;)

Reply:
UX/UI already take care of this and it passes WCAG level 2 accessibility criteria.
So I don't know why this 2px border is not accessible?

Reply:
I was talking about css standard not about accessibility on that point, look good to me thanks :)